app/util/metrics.ts

```diff
@@ -97,13 +96,18 @@ function constructBboxFromOperation(operation: DataOperation): BboxMetric {
  *
  * @returns the request metric
  */
-export function getRequestMetric(operation: DataOperation, serviceName: string): RequestMetric {
+export function getRequestMetric(
+  req: HarmonyRequest, operation: DataOperation, serviceName: string,
+): RequestMetric {
   const rangeBeginDateTime = operation.temporal?.start;
   const rangeEndDateTime = operation.temporal?.end;
+  const headers = req?.headers;
+  const forwardedHeader = headers ? headers['x-forwarded-for'] as unknown as string : '';
```

Comment:
Does your editor complain about this? Mine lets me just do `headers['x-forwarded-for'] as string`.

Reply:
weird - mine was - let me double-check again